src/frontend/screens/Settings/components/EnableFsync.tsx

```diff
@@ -14,7 +15,7 @@ const EnableFsync = () => {
   const isLinux = platform === 'linux'
   const [enableFsync, setEnableFsync] = useSetting('enableFsync', false)
 
-  if (!isLinux || isLinuxNative) {
+  if (!isLinux || isLinuxNative || isWindows) {
```

Comment:
`!isLinux` already covers the os being windows or mac, no need to for extra conditional here